agents-api/agents/article_extractor/tests.py

```
@@ -2,35 +2,38 @@
     def test_valid_url_validation(self):
         """Test URL validation with valid URLs"""
         valid_urls = [
             "https://example.com",
             "http://test.com/article",
             "https://subdomain.example.com/path",
         ]
-        
+
         for url in valid_urls:
             # Should not return validation error
             result, error = self.service.extract_links_from_article(url)
```

Ruff (0.13.3)

Unpacked variable `result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)